src/lib/useDataCollect.js

```diff
@@ -19,5 +27,54 @@ export default function useDataCollect() {
     collectSuggestClick: (obj) => fetcher(collectSuggestClick(obj)),
     collectRecommenderClick: (obj) => fetcher(collectRecommenderClick(obj)),
     collectSearchFeedback: (obj) => fetcher(collectSearchFeedback(obj)),
+    collectAddBookmark: async ({ title, materialType }) => {
+      matomoPushEvent([
+        "trackEvent",
+        "Huskeliste",
+        "Tilføj",
+        `${title} (${materialType})`,
+      ]);
+    },
```

Comment:
Forslag kunne være:
```

 matomoPushEvent([
        "Huskeliste",
        "Tilføj",
        `${title} (${materialType})`,
      ]);
```

Fandt den her med explainer til event parameterne. 
https://matomo.org/faq/reports/the-anatomy-of-an-event/